parser/parser.go

```diff
@@ -273,14 +275,25 @@ func (p *Parser) expectPeek(t token.Type) bool {
 	return false
 }
 
+// ErrorContext returns the current line and a pointer to the error position.
+// If prev is true, the error position is relative to the previous token instead of current one.
+func (p *Parser) ErrorContext(prev bool) string {
+	line, errPos := p.l.CurrentLine()
+	if prev {
+		errPos -= (p.l.Pos() - p.prevPos)
```

Comment:
This line require a comment as the computing you do is cryptic.

People don't read the function comment.

This line should be documented

Reply:
doing in #171  - note that if you don't read function comment... yet read code/non function comment... seems a bit odd / spurious argument

Reply:
I know 😬